heron/executor/src/python/heron-executor.py

```diff
@@ -465,7 +470,14 @@ def main():
   if len(sys.argv) != 32:
     print_usage()
     sys.exit(1)
-  executor = HeronExecutor(sys.argv)
+
+  # get the current environment and augment it with PEX_ROOT
+  shell_env = os.environ.copy()
+  shell_env["PEX_ROOT"] = os.path.join(os.path.abspath('.'), ".pex")
+  for k, v in shell_env.iteritems():
+    print k, v
```

Comment:
Was this print just for testing or do you want to ship this? If the latter it should be do_print and the print line should provide more context around what's being being printed.